More Rubocop tidying

hmallett opened the pull request:
Includes some of the non-default cops.

mvz commented:
Hi @hmallett there is an existing pull request from me in #593 to work on RuboCop offenses. Be careful not to avoid double work.

mvz commented:
Which non-default cops did you use? They are not enabled in `.rubocop.yml` in this pull request.

hmallett commented:
I have added the additional cops to rubocop.yml.

@mvz I was keeping an eye on your changes, to avoid duplication. I have rebased this on top of your changes.

mvz commented:
Looks like you rebased but then pulled your published branch again. If you want your rebase to work, you need to force push.

hmallett commented:
Sorted. Re-rebased and force pushed.
